README.md

```diff
@@ -113,7 +113,7 @@ Make sure to label the issues correctly and provide a proper explanation.
 Suggestions or feature requests can be labeled with "Enhancement", bug reports with "Bug", etc.
 
 ### Entity system
-The entity system (or colloquially known as "enemy sync") is a system that allows entities (enemies and bosses in the game) to be synchronised across clients.
+The entity system (or colloquially known as "enemy sync") is a system that allows entities (enemies and bosses in the game) to be synchronized across clients.
```

Comment:
This change was not necessary; both spellings were correct, albeit non-American vs American spelling.

Reply:
We can remove this one.